### ISSUE TITLE
JS error

### ISSUE THREAD

**Issue by Arkezis** [TFSExt/Forms#11]
Hi,

I've installed your plugin few weeks ago (but didn't use it much).
Unfortunately, it seems that it stopped working :'(

![image](https://user-images.githubusercontent.com/2463222/49951460-46542980-fefa-11e8-855c-1fd6aa58add5.png)

Any idae ? :(

**Comment by boyko-ant** [comment]
Hi!
Thanks, I will check that out. Just to double check - are you using Azure DevOps (aka VSTS), or you are using on-premise version?

**Comment by Arkezis** [comment]
Hi, thanks for the fast answer ! 
Using Azure DevOps here :)

**Comment by Arkezis** [comment]
Now, it's what I got : 

> Linked Wiki pages by Anton Boyko is taking longer than expected to load.Learn More about this extension, including available support options.

:(

**Comment by boyko-ant** [comment]
Hi!
I have checked it several times from different accounts. It seems like it was an issue with Microsoft CDN. Currently everything works fine for me.

**Comment by Arkezis** [comment]
Bouarf, it's what I feared :(
I'm in France, maybe an issue about this... 
I believe you're not in France, right ?

**Comment by boyko-ant** [comment]
No, I'm from Ukraine. But I can try to create a VM inside French DC and see how it will work from there.

**Comment by Arkezis** [comment]
Hi,
Did you have time to investigate on this from Ukraine ? :)
It seems that sometimes, it works and 5 minutes later, it goes with the 404 error :(
